### PR TITLE
integrate aliyun cli with gardenctl

### LINKED ISSUE #68: Aliyun CLI support
`gardenctl` has support for the different infrastructure CLIs, all but the latest addition, `aliyun`. Could this be added as well, please (it's so handy, when necessary)?

cc: @jia-jerry, Minchao Wang, and Emoin Lanyu

### PR DESCRIPTION
**Pull request by EmoinLanyu**
**What this PR does / why we need it**:
Integrate aliyun infrastructure cli with gardenctl.

**Which issue(s) this PR fixes**:
Fixes #68 

**Special notes for your reviewer**:
cc Jerry, Minchao

**Release note**:

```improvement operator
'gardenctl aliyun <args>' is now available on alicloud clusters.
```

### CODE REVIEW

**Comment by vlerenc** [comment]
Wow, that was fast, thank you!

File: cmd/operate.go [hunk 1]
```diff
@@ -43,6 +43,22 @@ func removeOldEntryAWS(filePath, contains string) {
 	checkError(err)
 }
 
+// removeOldEntryAliyun removes old aliyun credential entry for gardenctl if existing
+// ** should this be merged with the AWS one?
```
Comment (DockToFuture) [inline cmd/operate.go]: Please merge this with the aws func and make one generic function out of it

**Comment by EmoinLanyu** [comment]
Hi Sebastian and Jerry,

I have merged the functions and modified the scripts as per your request. Please check if this is okay for you, thx!

**Comment by DockToFuture** [comment]
> Hi Sebastian and Jerry,
> 
> I have merged the functions and modified the scripts as per your request. Please check if this is okay for you, thx!

LGTM, please squash the commits

**Comment by EmoinLanyu** [comment]
> > Hi Sebastian and Jerry,
> > I have merged the functions and modified the scripts as per your request. Please check if this is okay for you, thx!
> 
> LGTM, please squash the commits

done :)

**Pull request merged**